src/routes/dashboardRoute.ts

```diff
@@ -0,0 +1,49 @@
+import { dashbaordService } from "../services/dashboardService";
+import express from 'express';
+import { dashboardPermmisionsService } from "../services/dashboardPermmisionsService";
+
+const router = express.Router();
+
+router.post('/create', async (req, res) => {
+    try {
+        const dashboardName = req.body.dashboard.dashboardName;
+        const id = await dashbaordService.createNewDashbord(dashboardName);
+        res.send(id);
+    } catch (error) {
+        res.status(400).send(error);
```

Comment:
we should look into creating an error middleware, we got one in our office counterpart ;)

Reply:
added cooment